src/Frontend/Modules/FormBuilder/Widgets/Form.php

```diff
@@ -152,7 +152,6 @@ private function loadForm()
                 $item['name'] = 'field' . $field['id'];
                 $item['type'] = $field['type'];
                 $item['label'] = (isset($field['settings']['label'])) ? $field['settings']['label'] : '';
-                $item['placeholder'] = (isset($field['settings']['placeholder']) ? $field['settings']['placeholder'] : null);
```

Comment:
I'm guessing it's some reverse mechanisme from git because of a merge. Maybe I did a bad merge from master. Can't say honestly. The issue was that my original start from master branch was out of date. When I merge to a newer master for testing all those commits are attached to my project in a merge commit. Removing them fix would this problem.